from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Frecuencias(object):
    def setupUi(self, Form):
        Form.setObjectName("Form")
        Form.resize(1122, 648)
        self.verticalLayout = QtWidgets.QVBoxLayout(Form)
        self.verticalLayout.setObjectName("verticalLayout")
        self.groupBox = QtWidgets.QGroupBox(Form)
        self.groupBox.setFlat(True)
        self.groupBox.setObjectName("groupBox")
        self.gridLayout = QtWidgets.QGridLayout(self.groupBox)
        self.gridLayout.setObjectName("gridLayout")
        self.label_2 = QtWidgets.QLabel(self.groupBox)
        self.label_2.setObjectName("label_2")
        self.gridLayout.addWidget(self.label_2, 0, 1, 1, 1)
        self.label_6 = QtWidgets.QLabel(self.groupBox)
        self.label_6.setObjectName("label_6")
        self.gridLayout.addWidget(self.label_6, 2, 2, 1, 1)
        self.tabla_fre_act3 = QtWidgets.QTableWidget(self.groupBox)
        self.tabla_fre_act3.setObjectName("tabla_fre_act3")
        self.tabla_fre_act3.setColumnCount(3)
        self.tabla_fre_act3.setRowCount(4)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act3.setVerticalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act3.setVerticalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act3.setVerticalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act3.setVerticalHeaderItem(3, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act3.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act3.setHorizontalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act3.setHorizontalHeaderItem(2, item)
        self.gridLayout.addWidget(self.tabla_fre_act3, 1, 2, 1, 1)
        self.tabla_fre_act6 = QtWidgets.QTableWidget(self.groupBox)
        self.tabla_fre_act6.setObjectName("tabla_fre_act6")
        self.tabla_fre_act6.setColumnCount(3)
        self.tabla_fre_act6.setRowCount(4)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act6.setVerticalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act6.setVerticalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act6.setVerticalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act6.setVerticalHeaderItem(3, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act6.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act6.setHorizontalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act6.setHorizontalHeaderItem(2, item)
        self.gridLayout.addWidget(self.tabla_fre_act6, 3, 2, 1, 1)
        self.tabla_fre_act2 = QtWidgets.QTableWidget(self.groupBox)
        self.tabla_fre_act2.setObjectName("tabla_fre_act2")
        self.tabla_fre_act2.setColumnCount(3)
        self.tabla_fre_act2.setRowCount(4)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act2.setVerticalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act2.setVerticalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act2.setVerticalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act2.setVerticalHeaderItem(3, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act2.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act2.setHorizontalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act2.setHorizontalHeaderItem(2, item)
        self.gridLayout.addWidget(self.tabla_fre_act2, 1, 1, 1, 1)
        self.tabla_fre_act5 = QtWidgets.QTableWidget(self.groupBox)
        self.tabla_fre_act5.setObjectName("tabla_fre_act5")
        self.tabla_fre_act5.setColumnCount(3)
        self.tabla_fre_act5.setRowCount(4)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act5.setVerticalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act5.setVerticalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act5.setVerticalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act5.setVerticalHeaderItem(3, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act5.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act5.setHorizontalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act5.setHorizontalHeaderItem(2, item)
        self.gridLayout.addWidget(self.tabla_fre_act5, 3, 1, 1, 1)
        spacerItem = QtWidgets.QSpacerItem(10, 15, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Minimum)
        self.gridLayout.addItem(spacerItem, 4, 2, 1, 1)
        self.label_5 = QtWidgets.QLabel(self.groupBox)
        self.label_5.setObjectName("label_5")
        self.gridLayout.addWidget(self.label_5, 2, 1, 1, 1)
        self.label = QtWidgets.QLabel(self.groupBox)
        self.label.setObjectName("label")
        self.gridLayout.addWidget(self.label, 0, 0, 1, 1)
        self.tabla_fre_act4 = QtWidgets.QTableWidget(self.groupBox)
        self.tabla_fre_act4.setObjectName("tabla_fre_act4")
        self.tabla_fre_act4.setColumnCount(3)
        self.tabla_fre_act4.setRowCount(4)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act4.setVerticalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act4.setVerticalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act4.setVerticalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act4.setVerticalHeaderItem(3, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act4.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act4.setHorizontalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act4.setHorizontalHeaderItem(2, item)
        self.gridLayout.addWidget(self.tabla_fre_act4, 3, 0, 1, 1)
        self.label_4 = QtWidgets.QLabel(self.groupBox)
        self.label_4.setObjectName("label_4")
        self.gridLayout.addWidget(self.label_4, 2, 0, 1, 1)
        self.label_3 = QtWidgets.QLabel(self.groupBox)
        self.label_3.setObjectName("label_3")
        self.gridLayout.addWidget(self.label_3, 0, 2, 1, 1)
        self.tabla_fre_act1 = QtWidgets.QTableWidget(self.groupBox)
        self.tabla_fre_act1.setObjectName("tabla_fre_act1")
        self.tabla_fre_act1.setColumnCount(3)
        self.tabla_fre_act1.setRowCount(4)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act1.setVerticalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act1.setVerticalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act1.setVerticalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act1.setVerticalHeaderItem(3, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act1.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act1.setHorizontalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tabla_fre_act1.setHorizontalHeaderItem(2, item)
        self.gridLayout.addWidget(self.tabla_fre_act1, 1, 0, 1, 1)
        self.verticalLayout.addWidget(self.groupBox)

        self.retranslateUi(Form)
        QtCore.QMetaObject.connectSlotsByName(Form)

    def retranslateUi(self, Form):
        _translate = QtCore.QCoreApplication.translate
        Form.setWindowTitle(_translate("Form", "Form"))
        self.groupBox.setTitle(_translate("Form", "Tablas de Frecuencia"))
        self.label_2.setText(_translate("Form", "Actividad 2"))
        self.label_6.setText(_translate("Form", "Actividad 6"))
        item = self.tabla_fre_act3.verticalHeaderItem(0)
        item.setText(_translate("Form", "A"))
        item = self.tabla_fre_act3.verticalHeaderItem(1)
        item.setText(_translate("Form", "NP"))
        item = self.tabla_fre_act3.verticalHeaderItem(2)
        item.setText(_translate("Form", "R"))
        item = self.tabla_fre_act3.verticalHeaderItem(3)
        item.setText(_translate("Form", "Total"))
        item = self.tabla_fre_act3.horizontalHeaderItem(0)
        item.setText(_translate("Form", "NO"))
        item = self.tabla_fre_act3.horizontalHeaderItem(1)
        item.setText(_translate("Form", "SI"))
        item = self.tabla_fre_act3.horizontalHeaderItem(2)
        item.setText(_translate("Form", "Total"))
        item = self.tabla_fre_act6.verticalHeaderItem(0)
        item.setText(_translate("Form", "A"))
        item = self.tabla_fre_act6.verticalHeaderItem(1)
        item.setText(_translate("Form", "NP"))
        item = self.tabla_fre_act6.verticalHeaderItem(2)
        item.setText(_translate("Form", "R"))
        item = self.tabla_fre_act6.verticalHeaderItem(3)
        item.setText(_translate("Form", "Total"))
        item = self.tabla_fre_act6.horizontalHeaderItem(0)
        item.setText(_translate("Form", "NO"))
        item = self.tabla_fre_act6.horizontalHeaderItem(1)
        item.setText(_translate("Form", "SI"))
        item = self.tabla_fre_act6.horizontalHeaderItem(2)
        item.setText(_translate("Form", "Total"))
        item = self.tabla_fre_act2.verticalHeaderItem(0)
        item.setText(_translate("Form", "A"))
        item = self.tabla_fre_act2.verticalHeaderItem(1)
        item.setText(_translate("Form", "NP "))
        item = self.tabla_fre_act2.verticalHeaderItem(2)
        item.setText(_translate("Form", "R"))
        item = self.tabla_fre_act2.verticalHeaderItem(3)
        item.setText(_translate("Form", "Total"))
        item = self.tabla_fre_act2.horizontalHeaderItem(0)
        item.setText(_translate("Form", "NO"))
        item = self.tabla_fre_act2.horizontalHeaderItem(1)
        item.setText(_translate("Form", "SI"))
        item = self.tabla_fre_act2.horizontalHeaderItem(2)
        item.setText(_translate("Form", "Total"))
        item = self.tabla_fre_act5.verticalHeaderItem(0)
        item.setText(_translate("Form", "A"))
        item = self.tabla_fre_act5.verticalHeaderItem(1)
        item.setText(_translate("Form", "NP"))
        item = self.tabla_fre_act5.verticalHeaderItem(2)
        item.setText(_translate("Form", "R"))
        item = self.tabla_fre_act5.verticalHeaderItem(3)
        item.setText(_translate("Form", "Total"))
        item = self.tabla_fre_act5.horizontalHeaderItem(0)
        item.setText(_translate("Form", "NO"))
        item = self.tabla_fre_act5.horizontalHeaderItem(1)
        item.setText(_translate("Form", "SI"))
        item = self.tabla_fre_act5.horizontalHeaderItem(2)
        item.setText(_translate("Form", "Total"))
        self.label_5.setText(_translate("Form", "Actividad 5"))
        self.label.setText(_translate("Form", "Actividad 1"))
        item = self.tabla_fre_act4.verticalHeaderItem(0)
        item.setText(_translate("Form", "A"))
        item = self.tabla_fre_act4.verticalHeaderItem(1)
        item.setText(_translate("Form", "NP"))
        item = self.tabla_fre_act4.verticalHeaderItem(2)
        item.setText(_translate("Form", "R"))
        item = self.tabla_fre_act4.verticalHeaderItem(3)
        item.setText(_translate("Form", "Total"))
        item = self.tabla_fre_act4.horizontalHeaderItem(0)
        item.setText(_translate("Form", "NO"))
        item = self.tabla_fre_act4.horizontalHeaderItem(1)
        item.setText(_translate("Form", "SI"))
        item = self.tabla_fre_act4.horizontalHeaderItem(2)
        item.setText(_translate("Form", "Total"))
        self.label_4.setText(_translate("Form", "Actividad 4"))
        self.label_3.setText(_translate("Form", "Actividad 3"))
        item = self.tabla_fre_act1.verticalHeaderItem(0)
        item.setText(_translate("Form", "A"))
        item = self.tabla_fre_act1.verticalHeaderItem(1)
        item.setText(_translate("Form", "NP"))
        item = self.tabla_fre_act1.verticalHeaderItem(2)
        item.setText(_translate("Form", "R"))
        item = self.tabla_fre_act1.verticalHeaderItem(3)
        item.setText(_translate("Form", "Total"))
        item = self.tabla_fre_act1.horizontalHeaderItem(0)
        item.setText(_translate("Form", "NO"))
        item = self.tabla_fre_act1.horizontalHeaderItem(1)
        item.setText(_translate("Form", "SI"))
        item = self.tabla_fre_act1.horizontalHeaderItem(2)
        item.setText(_translate("Form", "Total"))
